packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -78,8 +78,9 @@ const getIsOutOfStock = (product, optionCodes, optionSelections) => {
             optionSelections,
             variants
         });
+        const stockStatus = item?.product?.stock_status;
 
-        return item.product.stock_status === OUT_OF_STOCK_CODE;
+        return stockStatus === OUT_OF_STOCK_CODE || !stockStatus;
```

Comment:
Not sure this `||` gate is needed, `null | undefined === 'OUT_OF_STOCK'` should still be `false`.

```suggestion
        return stockStatus === OUT_OF_STOCK_CODE;
```

Reply:
If stockStatus is `null | undefined` we want to return `true`, so `|| !stockStatus` is necessary unless there is a more elegant way to write it